functions/schedule-notifications.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+const functions = require('firebase-functions');
+const admin = require('firebase-admin');
+const moment = require('moment');
+
+const FORMAT = 'HH:mm';
+const BASE_URL = 'https://hoverboard-v2-dev.firebaseapp.com';
```

Comment:
better move it to config also

Reply:
It's not finished, working on another solution

Reply:
ok